examples/basic.js

```diff
@@ -66,6 +57,34 @@ var payload = {
 };
 
 console.log("Sending payload", payload);
+
+/**
+ * Since maxBatchCount is set to 1, calling send
+ * will immediately send the payload.
+ * 
+ * The underlying HTTP POST request is made to
+ *
+ *     https://localhost:8088/services/collector/event/1.0
+ *
+ * with the following body
+ *
+ *     {
+ *         "metadata": {
```

Comment:
This definitely does not look right. There is no `metadata` object in the HTTP Input - those keys should be in the top level object, adjacent to `event`.